### PR TITLE
Feature/soc7 nostack services

### PR DESCRIPTION
**Pull request by dmpop**
Devs, please review the Aodh and NTP barclamps parts. Thanks!

### CODE REVIEW

File: xml/depl_nostack_services.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    </listitem>
+   </varlistentry>
+    <varlistentry>
+    <term><guimenu>ntp-client</guimenu>
+    </term>
+    <listitem>
+     <para>
+      The <systemitem>ntp-client</systemitem> role can be assigned to any
+      node. Nodes with the ntp-client role assigned to them keep their time in
+      sync using NTP server in your cloud.
```
Comment (bmwiedemann) [inline xml/depl_nostack_services.xml]: using NTP server**s**
or
using **an** NTP server

File: xml/depl_nostack_services.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  <variablelist>
+   <varlistentry>
+    <term><guimenu>ntp-server</guimenu>
+    </term>
+    <listitem>
+     <para>
+      A node that acts as an NTP server for NTP clients in your cloud. There
+      can be more than one node with the ntp-server role in your cloud. In
+      this scenario, the NTP server nodes can communicate with each other to
+      keep their time in sync.
```
Comment (bmwiedemann) [inline xml/depl_nostack_services.xml]: with each other **and the specified external servers**

Comment (dmpop) [inline xml/depl_nostack_services.xml]: @bmwiedemann Thank you very much for your feedback! I'll update the text as per your suggestions.

File: xml/depl_nostack_services.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  </dm:docmanager>
+ </info>
+  <para>
+    In addition to &ostack; &barcl;s, &suse; &ostack; Cloud includes several components that can be configured using the appropriate  &crow; &barcl;s.
+   </para>
+    <sect1 xml:id="sec.depl.nostack.ntp">
+  <title>Configuring the NTP Service</title>
+  <para>
+    The NTP service is responsible for keeping the clocks in your cloud servers
+    in sync. The NTP component is deployed on the Administration Server automatically using the default settings. The NTP &barcl; can be used to specify IP addresses of the external NTP servers and assign specific roles to the desired nodes. The following parameter can be configured using the NTP &barcl;:
```
Comment (bmwiedemann) [inline xml/depl_nostack_services.xml]: Maybe mention here, that without synchronized clocks, chef-client will not work

Comment (dmpop) [inline xml/depl_nostack_services.xml]: How about something like this:

*Among other things, synchronized clocks ensure that the chef-client works properly.*

Comment (bmwiedemann) [inline xml/depl_nostack_services.xml]: sounds good. And what about *making it easier to read logs coming from different nodes*
(about correlating timestamps in logs)

Comment (dmpop) [inline xml/depl_nostack_services.xml]: Sure! I'll add that, too.

**Pull request closed without merging**